nvflare/tool/poc/poc_commands.py

```diff
@@ -218,10 +221,21 @@ def get_fl_server_name(project_config: OrderedDict) -> str:
         raise CLIException(f"project should only have one server, but {len(servers)} are provided: {servers}")
 
 
-def get_proj_admin(project_config: OrderedDict):
+def get_fl_admins(project_config: OrderedDict, is_project_admin: bool):
     participants: List[dict] = project_config["participants"]
-    admins = [p["name"] for p in participants if p["type"] == "admin"]
+    return [
+        p["name"]
+        for p in participants
+        if p["type"] == "admin" and (p["role"] == "project_admin" if is_project_admin else p["role"] != "project_admin")
+    ]
+
+
+def get_other_admins(project_config: OrderedDict):
+    return get_fl_admins(project_config, is_project_admin=False)
 
+
+def get_proj_admin(project_config: OrderedDict):
+    admins = get_fl_admins(project_config, is_project_admin=True)
     if len(admins) == 1:
         return admins[0]
```

Comment:
we can only have 1 project admins? @IsaacYangSLA

Reply:
the else condition say:
```
else:
    raise CLIException(f"project should only have only one project admin, but {len(admins)} are provided: {admins}")
``` 
        
 ^ is this statement true?

Reply:
There should be only ONE project_admin allowed through the provision tool. @IsaacYangSLA will add the enforcement in the future.